src/netflix/Main.java

```diff
@@ -0,0 +1,144 @@
+package netflix;
+import java.io.*;
+import java.util.*;
+import javax.print.attribute.standard.MediaName;
+
+public class Main {
+    public static void main(String[] args) throws IOException{
+
+        // bufferedreader for user input
+        BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
+
+        // instance variables
+        String strLogoff;
+        String strNickname;
+        boolean blnMaturity;
+        int intAge;
+        boolean end = false;
+        String str = "";
+
+        // create objects for user and list 
+        User newuser = new User("nickname", 0, false);
+        MyList list = new MyList();
+
+        System.out.println("Welcome to Netflix Recommendations!\n \nIf you decide to logoff, please type 'LOGOFF' \nTo start, please type in 'START'\n");
+
+        // loop the program
+        while (!str.toUpperCase().equals("LOGOFF")) {
+    
+            str = input.readLine().toUpperCase();
+
+            // logoff option
+            if (str.equals("LOGOFF")){
+                System.out.println("Thank you for using Netflix Recommendations");
+                break;
+            }
+
+            // Welcome page for user input 
+            System.out.println("\nWhat is your nickname? ");
+            newuser.setNickname();
+
+            System.out.print("What is your age? ");
+            newuser.setAge();
+
+            if (str.equals("START")) {
+
+                // check for user age to display appropriate shows
+                if (newuser.getAge() >= 18) {
+                    blnMaturity = true;
+                    System.out.println("You have access to all the content.");
+                    System.out.println();
+                    System.out.println(list.getMovieList());
+                    System.out.println(list.getShowList());
+                    System.out.println("Type 'recommendations' for recommeded media.");
+
+                    str = input.readLine().toUpperCase();
+
+                    // display recommendations
+                    if (str.equals("RECOMMENDATIONS")) {
+                        System.out.println("\nWhat genre do you like? [HORROR] [COMEDY] [ACTION] [FANTASY] [ADVENTURE]");
+                        String genre = input.readLine().toUpperCase();
+
+                        if (genre.equals("HORROR")) {
+                            System.out.println("The Walking Dead");
```

Comment:
Recommendations should not be hardcoded in.  The information exist within the MyList, hardcoding it negates the purpose of MyList.  The info for movies/shows should be maintained in one place